samples/struct.js

```diff
@@ -0,0 +1,431 @@
+/**
+ * Copyright 2018, Google LLC
+ * Licensed under the Apache License, Version 2.0(the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+function writeDataForStructQueries(instanceId, databaseId, projectId) {
+  // [START spanner_write_data_for_struct_queries]
+  // Imports the Google Cloud client library
+  const Spanner = require('@google-cloud/spanner');
+
+  /**
+   * TODO(developer): Uncomment and update the following lines before running the sample.
+   */
+  // const projectId = 'my-project-id';
+  // const instanceId = 'my-instance';
+  // const databaseId = 'my-database';
+
+  // Creates a client
+  const spanner = new Spanner({
+    projectId: projectId,
+  });
+
+  // Gets a reference to a Cloud Spanner instance and database
+  const instance = spanner.instance(instanceId);
+  const database = instance.database(databaseId);
+
+  // Instantiate Spanner table objects
+  const singersTable = database.table('Singers');
+
+  // Inserts rows into the Singers table
+  // Note: Cloud Spanner interprets Node.js numbers as FLOAT64s, so
+  // they must be converted to strings before being inserted as INT64s
+  singersTable
+    .insert([
+      {
+        SingerId: '6',
+        FirstName: 'Elena',
+        LastName: 'Campbell',
+      },
+      {
+        SingerId: '7',
+        FirstName: 'Gabriel',
+        LastName: 'Wright',
+      },
+      {
+        SingerId: '8',
+        FirstName: 'Benjamin',
+        LastName: 'Martinez',
+      },
+      {
+        SingerId: '9',
+        FirstName: 'Hannah',
+        LastName: 'Harris',
+      },
+    ])
+    .then(() => {
+      console.log('Inserted data.');
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    })
+    .then(() => {
```

Comment:
You could shorten to 

`.then( () => database.close())`

If the function body is a single statement, return is implicit.

Reply:
Done